cardano-cli/src/Cardano/CLI/Byron/UpdateProposal.hs

```diff
@@ -141,10 +141,11 @@ createUpdateProposal
 createUpdateProposal nw sKey pVer sVer sysTag inshash paramsToUpdate =
     signProposal (toByronProtocolMagicId nw) proposalBody noPassSigningKey
   where
-    proposalBody = ProposalBody pVer protocolParamsUpdate sVer metaData
+    proposalBody = ProposalBody pVer protocolParamsUpdate sVer updateMetadata
+
+    updateMetadata :: M.Map SystemTag InstallerHash
+    updateMetadata = M.singleton sysTag inshash
 
-    metaData :: M.Map SystemTag InstallerHash
-    metaData = M.singleton sysTag inshash
```

Comment:
Rename `metaData` to `metadata` gives a name shadowing warning, so I picked something else (`metaData` would be wrong, IMO).

Reply:
`metadatum`?